dev-util/bindgen/metadata.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+    <maintainer type="project">
+        <email>rust@gentoo.org</email>
+    </maintainer>
+    <upstream>
+        <remote-id type="github">rust-lang/rust-bindgen</remote-id>
```

Comment:
add

```xml
<changelog>https://github.com/rust-lang/rust-bindgen/blob/master/CHANGELOG.md</changelog>
```

here as well, and also throw in kernel team as second maint

```xml
<maintainer type="project">
  <email>kernel@gentoo.org</email>
  <name>Gentoo Kernel Project</name>
</maintainer>
```

Reply:
Fair enough, addressed in next revision